src/ml_tooling/search/gridsearch.py

```diff
@@ -0,0 +1,27 @@
+from sklearn import clone
+from sklearn.model_selection import ParameterGrid
+import joblib
+
+
+def _fit_gridpoint(estimator, params, train_x, train_y):
+    estimator.set_params(**params)
+    estimator.fit(train_x, train_y)
+    return estimator
+
+
+def gridsearch(estimator, train_x, train_y, params, n_jobs):
```

Comment:
Firstly it should not be called `gridsearch` as it doesn't do the gridsearch but only generates the estimators so should probabily be name  `make_estimators_for_gridsearch`. Secondly, why do we train it, when it is trained again on the folds when the gridsearch is done?

Reply:
Good point, was originally done for non-CV metric, but of course cross_val_score already scores the model. I assume we would never allow for a non-CV gridsearch?

Reply:
Should be fixed now